### PR TITLE
fix(get/set): Add support for global configuration.

### PR DESCRIPTION
**Pull request by hansl**
If we dont find the local one, we error now. If we get/set with --global, we properly do the right thing.

### CODE REVIEW

File: packages/angular-cli/ember-cli/lib/tasks/npm-install.js [hunk 1]
```diff
@@ -1,11 +1,64 @@
 'use strict';
```
Comment (Brocco) [inline packages/angular-cli/ember-cli/lib/tasks/npm-install.js]: Changes to npm-install should be part of a separate PR

Comment (hansl) [inline packages/angular-cli/ember-cli/lib/tasks/npm-install.js]: Yes, done.

**Comment by hansl** [comment]
Investigating tests and e2e.
